nvm.sh

```diff
@@ -1848,13 +1848,14 @@ nvm_install_source() {
 
   local make
   make='make'
-  if [ "${NVM_OS}" = 'freebsd' ]; then
-    make='gmake'
-    MAKE_CXX='CXX=c++'
-  elif [ "${NVM_OS}" = 'aix' ]; then
+  if [ "${NVM_OS}" = 'freebsd' ] || [ "${NVM_OS}" = 'aix' ]; then
```

Comment:
I prefer to keep all the conditions for each OS separate - ie, i think it's more maintainable to repeat the `make` assignment rather than have multiple confusing code paths for the same OS.